src/images/logos/eodev-mobile.svg

```diff
@@ -2,6 +2,7 @@
   <style>
     .st0{fill:#FFFFFF;} .st1{fill:#020A1B;}
   </style>
+  <title>eodev mobile logo</title>
```

Comment:
I think we can skip "mobile". this is for page visitors - I think that they won't care if logo is mobile or not

Reply:
fixed!